pkg/secrets/factory.go

```diff
@@ -105,7 +105,7 @@ func readPasswordStdin() ([]byte, error) {
 	printPasswordPrompt()
 	password, err := term.ReadPassword(int(os.Stdin.Fd()))
 	// Start new line after receiving password to ensure errors are printed correctly.
-	logger.Infof("")
+	fmt.Println()
```

Comment:
Deliberately using `fmt.Println()` here - this code is only ever called in a terminal context, and we need to put a linefeed, not print an empty log line (which will have a timestamp and log level)